src/components/portrait/portrait.style.tsx

```diff
@@ -13,7 +13,7 @@ type StylingForSize = {
 };
 
 function stylingForSize({ size, initials }: StylingForSize) {
-  const params = PORTRAIT_SIZE_PARAMS[size];
+  const params = size ? PORTRAIT_SIZE_PARAMS[size] : undefined;
```

Comment:
```suggestion
  const params = PORTRAIT_SIZE_PARAMS[size];
```
This should probably work as `size` type can't be undefined. Setting this as undefined could effect the return of the function as it's expecting a value to make a px value.